src/DotNet/Importer.cs

```diff
@@ -508,6 +516,8 @@ public Importer(ModuleDef module, ImporterOptions options, GenericParamContext g
 			}
 		}
 
+		bool IsThisModule(Module module2) => module.Name == module2.Name && IsThisAssembly(module2);
```

Comment:
`SigComparer` uses `module2.ScopeName` and it should be case insensitive comparison, see `bool Equals(IModule a, Module b) {`.

Reply:
Done.